sci-libs/arrayfire/arrayfire-9999.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit cmake-utils eutils git-r3
+
+DESCRIPTION="A general purpose GPU library."
+HOMEPAGE="http://www.arrayfire.com/"
+EGIT_REPO_URI="https://github.com/arrayfire/arrayfire.git"
+KEYWORDS="~amd64"
+
+LICENSE="ArrayFire"
+SLOT="0"
+IUSE="+examples +cpu cuda test"
+
+RDEPEND="
+	>=sys-devel/gcc-4.7.3-r1
+	virtual/blas
+	virtual/cblas
+	cuda? ( >=dev-util/nvidia-cuda-toolkit-6.0 )
+	sci-libs/fftw:3.0"
+DEPEND="${RDEPEND}
+	test? ( dev-vcs/subversion )"
```

Comment:
arrayfire fetches a svn repo for to googletests target. How should we handle this, if fetching during the build is not allowed?